redbot/core/utils/menus.py

```diff
@@ -224,3 +233,442 @@ async def task():
+        "\N{BLACK RIGHT-POINTING TRIANGLE}\ufe0f",
+        position=_dpy_menus.Last(0),
+        skip_if=_skip_single_arrows,
+    )
+    async def go_to_next_page(self, payload):
+        """go to the next page"""
+        await self.show_checked_page(self.current_page + 1)
+
+    @_dpy_menus.button(
+        "\N{BLACK LEFT-POINTING DOUBLE TRIANGLE WITH VERTICAL BAR}\ufe0f",
+        position=_dpy_menus.First(0),
+        skip_if=_skip_double_triangle_buttons,
+    )
+    async def go_to_first_page(self, payload):
+        """go to the first page"""
+        await self.show_page(0)
+
+    @_dpy_menus.button(
+        "\N{BLACK RIGHT-POINTING DOUBLE TRIANGLE WITH VERTICAL BAR}\ufe0f",
+        position=_dpy_menus.Last(1),
+        skip_if=_skip_double_triangle_buttons,
+    )
+    async def go_to_last_page(self, payload):
+        """go to the last page"""
+        # The call here is safe because it's guarded by skip_if
+        await self.show_page(self._source.get_max_pages() - 1)
+
+    @_dpy_menus.button("\N{CROSS MARK}")
+    async def stop_pages(self, payload: discord.RawReactionActionEvent) -> None:
+        """stops the pagination session."""
+        self.stop()
+
+
+class SimpleHybridMenu(HybridMenu, inherit_buttons=True):
+    def __init__(
+        self,
+        source: _dpy_menus.PageSource,
+        cog: Optional[commands.Cog] = None,
+        clear_reactions_after: bool = True,
+        delete_message_after: bool = True,
+        add_reactions: bool = True,
+        timeout: int = 180,
+        accept_keywords: bool = False,
+        **kwargs: Any,
+    ):
+        if accept_keywords:
+            keyword_to_reaction_mapping = {
+                _("last"): ["\N{BLACK RIGHT-POINTING DOUBLE TRIANGLE WITH VERTICAL BAR}\ufe0f"],
+                _("first"): ["\N{BLACK LEFT-POINTING DOUBLE TRIANGLE WITH VERTICAL BAR}\ufe0f"],
+                _("next"): ["\N{BLACK RIGHT-POINTING TRIANGLE}\ufe0f",],
+                _("previous"): ["\N{BLACK LEFT-POINTING TRIANGLE}\ufe0f",],
+                _("prev"): ["\N{BLACK LEFT-POINTING TRIANGLE}\ufe0f",],
+                _("close"): ["\N{CROSS MARK}"],
+            }
+        else:
+            keyword_to_reaction_mapping = None
+        super().__init__(
+            source=source,
+            cog=cog,
+            add_reactions=add_reactions,
+            timeout=timeout,
+            clear_reactions_after=clear_reactions_after,
+            delete_message_after=delete_message_after,
+            keyword_to_reaction_mapping=keyword_to_reaction_mapping,
+            **kwargs,
+        )
+
+
+class SimpleSource(_dpy_menus.ListPageSource):
+    def __init__(self, pages: Iterable[Union[List[str], List[discord.Embed]]]):
+        super().__init__(pages, per_page=1)
+
+    async def format_page(
+        self, menu: SimpleHybridMenu, page: Union[List[str], List[discord.Embed]]
+    ) -> Union[discord.Embed, str]:
+        """Sends thee specified page to the menu."""
+        return page
```

Comment:
These subclasses can't be added here. discord.py's menus aren't stable yet, placing them in this file would add a guarantee to them.